packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -82,6 +86,14 @@ const getPayloadOpts: ReqOpts = {routeId: "getPayload"};
  * https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.1/src/engine/interop/specification.md
  */
 export class ExecutionEngineHttp implements IExecutionEngine {
+  readonly emitter = new ExecutionEngineEventEmitter();
```

Comment:
Why do we need this emitter?

Reply:
For this use case. 

https://github.com/ChainSafe/lodestar/blob/df7a0c6650acf283ac9f0e5b7d18315a4c9b19a9/packages/beacon-node/src/node/nodejs.ts#L225-L237

Reply:
since we are not fcUing on state change as of now (as per discord discussions) may be we can remove it to kkeep techncal debt low